src/Config/RabbitMQWebJobsBuilderExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.RabbitMQ;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Hosting
+{
+    /// <summary>
+    /// Extension methods for RabbitMQ integration.
+    /// </summary>
+    public static class RabbitMQWebJobsBuilderExtensions
+    {
+        /// <summary>
+        /// Adds the RabbitMQ extension to the provided <see cref="IWebJobsBuilder"/>.
+        /// </summary>
+        /// <param name="builder">The <see cref="IWebJobsBuilder"/> to configure.</param>
+        public static IWebJobsBuilder AddRabbitMQ(this IWebJobsBuilder builder)
+        {
+            if (builder == null)
+            {
+                throw new ArgumentNullException(nameof(builder));
+            }
+
+            builder.AddExtension<RabbitMQExtensionConfigProvider>()
+                .ConfigureOptions<RabbitMQOptions>((config, path, options) =>
+                {
+                    options.Hostname = "localhost";

```

Comment:
Do not set options.Hostname and  options.QueueName. Assuming these are left over from testing :)